.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,7 @@
+-   id: pre-commit-all-the-way-down
+    name: pre-commit-all-the-way-down
```

Comment:
the name (and id) of the hook should be "python-doc" (or `python-rst`) to leave room for other similar hooks